js/src/components/app-select-control/index.js

```diff
@@ -12,18 +14,72 @@ import './index.scss';
 /**
  * Renders a `@wordpress/components`'s `SelectControl` with margin-bottom removed.
  *
- * If you provide `className` via props,
- * it will be added to the container div's `className`,
- * so that you can further control its style.
- *
- * @param {*} props
+ * @param {*} props The component props.
+ * @param {Array} props.options Array of options for the select dropdown. Each option should be an object containing `label` and `value` properties.
+ * @param {string} [props.className] Additional classname to further control the style of the component.
+ * @param {Function} [props.onChange=noop] Callback function triggered when the selected value changes. Receives the new value as an argument.
+ * @param {string} [props.value] The selected value. If no value is defined, the first option is selected and `onChange` is triggered when `autoSelectFirstOption` is true.
```

Comment:
How about adjusting this description to the following?

```suggestion
 * @param {string} [props.value] The currently selected value. This component should be used as a controlled component. A special case is that after mounting, when `autoSelectFirstOption` is true and `value` is undefined, it tries to call back `onChange` once to select the first option so that the `value` can be consistent with the `<select>` element's own value.
```

Reply:
👍